### PR TITLE
added bibl links

### PR DESCRIPTION
**Pull request by karindalziel**
closes issue https://github.com/CDRH/earlywashingtondc/issues/124

### CODE REVIEW

File: scripts/tei.p5.xsl [hunk 1]
```diff
@@ -36,6 +36,14 @@
 <!-- ==================================================================== -->
 <!--                            OVERRIDES                                 -->
 <!-- ==================================================================== -->
+  
+  <!-- Adding links to other cases via bibls https://github.com/CDRH/earlywashingtondc/issues/124 -->
+  
+  <xsl:template match="/TEI/text//bibl[@type='case'][@xml:id]">
+    <a href="{@xml:id}" class="tei_bibl_link">
```
Comment (jduss4) [inline scripts/tei.p5.xsl]: Does this work with relative links to the cases?  Or should we be subbing in a site-url for the first portion of the href?

Comment (karindalziel) [inline scripts/tei.p5.xsl]: I'm trying to think of any instance of one case linking to another where the URL structure would change. they are all this link format, correct? http://earlywashingtondc.org/cases/oscys.caseid.0251

Comment (jduss4) [inline scripts/tei.p5.xsl]: I think so, although presumably this will break on development where the url is `/earlywashingtondc/cases`

Comment (karindalziel) [reply scripts/tei.p5.xsl]: It would still work even if the URL changes. It's relative, not root.

Comment (karindalziel) [reply scripts/tei.p5.xsl]: Tested locally and it works. the doc links are also relative.

**Comment by jduss4** [comment]
Closed with https://github.com/CDRH/data_oscys/commit/db2fb8bb2c427638e338168ec458a982df4d970c

**Pull request closed without merging**